library/std/convert.qs

```diff
@@ -108,7 +136,7 @@ namespace Microsoft.Quantum.Convert {
     function ResultArrayAsBoolArray(input : Result[]) : Bool[] {
         mutable output = [];
         for r in input {
-            set output += [r == One];
+            set output += [ResultAsBool(r)];
```

Comment:
We prefer not to use library functions in other library functions unless necessary. This approach allows pieces of library code to be used separately, without the full context of the library. It also helps with learning and with AI-generated code.
Here, the call to ResultAsBool doesn't add much clarity or conciseness, but it would require a novice to check the description of that function. The previous code was self-explanatory. So I suggest we keep the old code.

Reply:
yep sure